Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -91,6 +91,37 @@ static media::OpenCdm::LicenseType webKitLicenseTypeToOpenCDM(CDMInstance::Licen
     }
 }
 
+class CDMInstanceOpenCDM : public CDMInstance {
```

Comment:
Should probably be final, I don't see this class removed from elsewhere in this patch either, so I think you'll need to rebase.